data/weapons.txt

```diff
@@ -457,6 +457,32 @@ effect "proton impact"
 	"velocity scale" -.005
 
 
+outfit "Proton Turret"
+	category "Turrets"
+	cost 260000
+	thumbnail "outfit/proton turret"
+	"mass" 36
+	"outfit space" -36
+	"weapon capacity" -36
+	"turret mounts" -1
+	weapon
+		sprite "projectile/proton"
+		"hardpoint sprite" "hardpoint/proton turret"
+		"hardpoint offset" 14.
+		sound "proton"
+		"hit effect" "proton impact" 3
+		"inaccuracy" 4
+		"turret turn" 2.5
+		"submunition" "Proton Fragment" 3
+		"velocity" 24
+		"lifetime" 2
+		"reload" 8
+		"firing energy" 15
+		"firing force" 2
+		"firing heat" 60
+	description "The Syndicate Proton Turret is a relatively new product designed to compete with Deep Sky's perennially-popular laser turrets for the anti-fighter defense market. Though a cruder weapon than its competition, the Proton Turret maintains a loyal following among captains dissatisfied with laser weapons' short range.
```

Comment:
`The Syndicate's Proton Turret is a relatively new product designed to compete with Deep Sky's perennially-popular laser turrets in the anti-fighter defense market. Though cruder than its competition, the Proton Turret maintains a loyal following among captains dissatisfied with laser weapons' short range.`

 - possessive `Syndicate`,
 - `for` -> `in`, 
 - ~~a~~ ~~weapon~~

Reply:
Also, `perennially-popular` -> `perennially popular`; -ly adverbs should not have hyphens.

Reply:
I'll do the last two. There's nothing wrong with "The Syndicate Proton Turret..."

Reply:
Indeed. It was more to parallel the possessive Deep Sky usage.